x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/dev_console.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo } from 'react';
```

Comment:
This is just temporary until "console" is added to the Endpoint list row actions